common4j/src/main/com/microsoft/identity/common/java/platform/AbstractDevicePopManager.java

```diff
@@ -1052,12 +1011,6 @@ private String mintSignedHttpRequestInternal(@Nullable final String httpMethod,
             errCode = INVALID_PROTECTION_PARAMS;
         }
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
-                && exception.getCause() instanceof KeyPermanentlyInvalidatedException) {
-            Logger.warn(TAG, "Unable to access asymmetric key - clearing.");
-            clearAsymmetricKey();
-        }
```

Comment:
This probably needs to be put back.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1683/commits/3d0b837f2723e94e18aa9787923b4ade0f72d0c7